client/src/public/ts/settings.ts

```
@@ -106,45 +113,91 @@ function getSettings(storageString: string) {
     }
   }
   console.log("Got settings!");
 }
 
 /**
  * Gets the stored settings from `localStorage`, then updates the client-side variables with the data.
+ * FIXME: I believe this just copies `getSettings`, I guess we can merge them? -mistertfy64
  * @param {string} storageString The stored string. This should be from `localStorage`.
  */
 function loadSettings(storageString: string) {
   let settings = JSON.parse(storageString);
```

Biome

[error] This let declares a variable that is only assigned once.

'settings' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)